io/src/FileSystem/private/GoogleCloudStorageReadableFile.cpp

```diff
@@ -178,7 +178,7 @@ return arrow::Status::OK();
 		//		results.GetResult().GetBody().read((char*)(out->get()->mutable_data()), bytesRead);
 
 		std::shared_ptr<arrow::ResizableBuffer> buffer;
-		AllocateResizableBuffer(arrow::default_memory_pool(), nbytes, &buffer);
+		AllocateResizableBuffer(nbytes, arrow::default_memory_pool());
```

Comment:
Should be something like this:
`buffer = AllocateResizableBuffer(nbytes, arrow::default_memory_pool());`
We need to make sure we are allocating the buffer

Reply:
Sure. thanks for the advice.